dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -72,7 +72,7 @@ HRESULT CFSDropTarget::_CopyItems(IShellFolder * pSFFrom, UINT cidl,
 
     TRACE ("(%p)->(%p,%u,%p)\n", this, pSFFrom, cidl, apidl);
 
-    STRRET strretFrom;
+    STRRET strretFrom = { 0 }; // Initialize strretFrom.pOleStr to NULL
     hr = pSFFrom->GetDisplayNameOf(NULL, SHGDN_FORPARSING, &strretFrom);
     if (FAILED_UNEXPECTEDLY(hr))
         return hr;
```

Comment:
Something I'm wondering, is what ensures that it is only the `pOleStr` member of the `STRRET strretFrom` variable that will be valid, below, and not one of the other members....
@learn-more Do you have a suggestion about that?

Reply:
Yes, that is a hack.
STRRET has an `uType` field that describes what member is actually used, and there are functions to extract the string from an STRRET.

Reply:
So the actual issue might be that some other field than `pOleStr` is filled by whatever is being called here, hence it makes sense to investigate the called code.

Reply:
@learn-more, could this serve as a temporary hack (since it stops the error message) until someone else can look into the details or until I can learn more about COM interfaces? (I think it will take me some time based on what I see now).

Reply:
I am not in favor of hiding an issue by working around it.
This debug print indicates that something is wrong, so why would we hide that?